agent/structs/connect_ca.go

```diff
@@ -102,8 +102,14 @@ type CARoot struct {
 	// active root.
 	RotatedOutAt time.Time `json:"-"`
 
-	// Type of private key used to create the CA cert.
+	// PrivateKeyType is the type of the private key used to sign certificates. It
+	// may be "rsa" or "ec". This is provided as a convenience to avoid parsing
+	// the public key from the certificate.
```

Comment:
```suggestion
	// the private key from the certificate.
```
Did you mean private?

Reply:
see below